### ISSUE TITLE
Unable to import @import '~@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';

### ISSUE THREAD

**Issue by Oryon86** [kuhnroyal/mat-datetimepicker#400]
Hello everyone,

I've a problem while importing the scss theme into my app : 

Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Can't find stylesheet to import.
  ╷
6 │ @import '@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
I saw the example on https://github.com/kuhnroyal/mat-datetimepicker/blob/release/src/styles.scss 

and i've seen this : 

// Use this import in a real project instead
//@import '~@mat-datetimepicker/core/datetimepicker/datetimepicker-theme.scss';
@import 'dist/core/datetimepicker/datetimepicker-theme.scss';

Did you test this import from node_modules directory?

Thank you for your response.

**Comment by bbortt** [comment]
sounds like #216.. does that help?

**Comment by Oryon86** [comment]
Tank you. It works fine with : 

```
"stylePreprocessorOptions": {
      "includePaths": [
           "node_modules"
       ]
 },
```

But it still a warning : 

`imports '~@angular/material/theming' with a tilde. Usage of '~' in imports is deprecated.`

**Comment by bbortt** [comment]
not so active in typescript development anymore, but I think that comes from the sass loader. looks like we should update the documentation on our side. looking at it (https://github.com/kuhnroyal/mat-datetimepicker#theming) the `stylePreprocessorOptions` is mentioned nowhere.

**Comment by gsontag** [comment]
With angular 15, support for tilde in import as been dropped, so `imports '~@angular/material/theming'` cause build to fail.

**Comment by bbortt** [comment]
closed since release `v11.0.0` is out.
